Python/compile.c

```diff
@@ -8783,7 +8788,9 @@ optimize_basic_block(struct compiler *c, basicblock *bb, PyObject *consts)
                     inst->i_opcode = NOP;
                     break;
                 }
-                i += swaptimize(bb, i);
+                if (swaptimize(bb, &i)) {
```

Comment:
```suggestion
                if (swaptimize(bb, &i) < 0) {
```
This is much common in the codebase.